Sources/Nimble/Matchers/Contain.swift

```diff
@@ -8,15 +8,15 @@ public func contain<S: Sequence, T: Equatable>(_ items: T...) -> Predicate<S>
 
 public func contain<S: Sequence, T: Equatable>(_ items: [T]) -> Predicate<S>
     where S.Iterator.Element == T {
-    return Predicate.fromDeprecatedClosure { actualExpression, failureMessage in
-        failureMessage.postfixMessage = "contain <\(arrayAsString(items))>"
+    return Predicate.simple("contain <\(arrayAsString(items))>") { actualExpression in
         if let actual = try actualExpression.evaluate() {
-            return items.all {
+            let matches = items.all {
```

Comment:
Not necessary in this PR, but wondering if we can rename this to `allSatisfy` and make it a compatibility shim for Swift < 4.2? https://github.com/apple/swift-evolution/blob/master/proposals/0207-containsOnly.md

Reply:
Totally makes sense 👍